pkgs/applications/misc/dwmbar/default.nix

```
@@ -0,0 +1,34 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation {
+  pname = "dwmbar";
+  version = "unstable-2021-12-22";
+
+  src = fetchFromGitHub {
+    owner = "thytom";
+    repo = "dwmbar";
+    rev = "574f5703c558a56bc9c354471543511255423dc7";
+    sha256 = "sha256-IrelZpgsxq2dnsjMdh7VC5eKffEGRbDkZmZBD+tROPs=";
+  };
+
+  installPhase = ''
+    mkdir -p $out/bin
+    mkdir -p $out/config
+    cp -r modules/ $out/config/
+    install config $out/config/config
+    install bar.sh $out/config/bar.sh
+    substituteInPlace dwmbar --replace 'DEFAULT_CONFIG_DIR="/usr/share/dwmbar"' "DEFAULT_CONFIG_DIR=\"$out/config\""
+    install dwmbar $out/bin/dwmbar
```

```suggestion
    cp -r modules/ $out/config/
    install config $out/config/config
    install bar.sh $out/config/bar.sh
    install dwmbar $out/bin/dwmbar
```
Please adjust the directories here. Also why are we not creating the directories with -d/-D on install instead of using mkdir?

---

@SuperSandro2000 Im very unfamiliar with the install command, but I believe I have implemented your suggestion. Let me know if there is anything I need to change. Thank you:)